cmd/ctr/commands/run/run_windows.go

```diff
@@ -64,6 +64,8 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 			opts = append(opts, oci.WithRootFSPath(""))
 		} else {
 			opts = append(opts, oci.WithDefaultSpec())
+			opts = append(opts, oci.WithWindowNetworksAllowUnqualifiedDNSQuery())
```

Comment:
Why remove these defaults? When don't we want them?

Reply:
In docker. Currently they are both dependent on what's being run.

Reply:
If we really think this is the right default then fine. I need to update CRI to pass these.